loopchain/rest_server/rest_server_rs.py

```diff
@@ -209,21 +209,21 @@ class Peer(HTTPMethodView):
 
             if peer_manager.peer_list:
                 async_futures: List[grpc.Future] = []
-                for peer_id in peer_manager.peer_list[conf.ALL_GROUP_ID]:
-                    async_future = ServerComponents().get_peer_status_async(peer_id, conf.ALL_GROUP_ID, channel)
+                for peer_id in peer_manager.peer_list:
+                    async_future = ServerComponents().get_peer_status_async(peer_id, "all_group_id", channel)
```

Comment:
This group ID is not deleted.

Reply:
`group_id` is partially removed here. Group ids related to grpc proto will be updated at the next pull request.